cdap-ui/webpack.config.cdap.js

```diff
@@ -150,7 +150,7 @@ var webpackConfig = {
   context: __dirname + '/app/cdap',
   entry: {
     // including babel-polyfill is temporary as of now. Once babel handles adding https://github.com/babel/babel/issues/4169.
-    'cdap': ['babel-polyfill', './cdap.js', 'rx', 'rx-dom']
+    'cdap': ['babel-polyfill', './cdap.js', 'rxjs']
```

Comment:
nit: now that we have combined both the `rx*` dependencies. Can we move `rxjs` to the vendor dll bundles? The reason we had it here is because the loading of `rx` and `rx-dom` was an issue in dll bundles.

Reply:
if we bundle rxjs in the dll bundle, then there is no point in doing selective import. I'm not sure why the build size increase. I haven't check that. Will take a look.

Reply:
I agree. We should check if treeshaking is possible, if not we should try moving that to the bundle as it can be cached for further requests.